src/SFML/Window/Win32/WindowImplWin32.hpp

```diff
@@ -262,19 +274,25 @@ class WindowImplWin32 : public WindowImpl
     static LRESULT CALLBACK globalOnEvent(HWND handle, UINT message, WPARAM wParam, LPARAM lParam);
 
     ////////////////////////////////////////////////////////////
+    // Type for storing a batch of dropped files
+    ////////////////////////////////////////////////////////////
+    typedef std::vector<String> FilesType;
+
+    ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    HWND     m_handle;           ///< Win32 handle of the window
-    LONG_PTR m_callback;         ///< Stores the original event callback function of the control
-    HCURSOR  m_cursor;           ///< The system cursor to display into the window
-    HICON    m_icon;             ///< Custom icon assigned to the window
-    bool     m_keyRepeatEnabled; ///< Automatic key-repeat state for keydown events
-    Vector2u m_lastSize;         ///< The last handled size of the window
-    bool     m_resizing;         ///< Is the window being resized?
-    Uint16   m_surrogate;        ///< First half of the surrogate pair, in case we're receiving a Unicode character in two events
-    bool     m_mouseInside;      ///< Mouse is inside the window?
-    bool     m_fullscreen;       ///< Is the window fullscreen?
-    bool     m_cursorGrabbed;    ///< Is the mouse cursor trapped?
+    HWND                 m_handle;           ///< Win32 handle of the window
+    LONG_PTR             m_callback;         ///< Stores the original event callback function of the control
+    HCURSOR              m_cursor;           ///< The system cursor to display into the window
+    HICON                m_icon;             ///< Custom icon assigned to the window
+    bool                 m_keyRepeatEnabled; ///< Automatic key-repeat state for keydown events
+    Vector2u             m_lastSize;         ///< The last handled size of the window
+    bool                 m_resizing;         ///< Is the window being resized?
+    Uint16               m_surrogate;        ///< First half of the surrogate pair, in case we're receiving a Unicode character in two events
+    bool                 m_mouseInside;      ///< Mouse is inside the window?
+    bool                 m_fullscreen;       ///< Is the window fullscreen?
+    bool                 m_cursorGrabbed;    ///< Is the mouse cursor trapped?
+    std::list<FilesType> m_droppedFiles;     ///< Buffer for the dropped files
```

Comment:
No, the event only stores a pointer to the buffer. See the first two bullet points in my issues list.